package/aioracle/provider_bridge/src/contract.rs

```diff
@@ -112,11 +116,12 @@ pub fn handle_update_service_contracts(
     service: String,
     contracts: Contracts,
 ) -> Result<HandleResponse, ContractError> {
-    let owner = OWNER.load(deps.storage)?;
-    if info.sender.ne(&owner) {
+    let mut service_info = get_service_info(deps.as_ref(), service.to_string()).unwrap();
```

Comment:
we should not use unwrap(). Instead, use ? to automatically handle errors. Unwrap will make the transaction panic, which is not good